.github/workflows/verifyCommitsSigned.yml

```
@@ -0,0 +1,135 @@
+  contents: read # required to read PR metadata via the REST API
+
+# Cancel any in-progress run for the same PR when a new commit is pushed.
+# The status check that lands on the PR will reflect the latest commit only.
+concurrency:
+  group: verify-commits-signed-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+
+jobs:
+  verify-signed-commits:
+    # TEMPORARY for live-test on this draft PR — will be reverted before merge.
+    if: ${{ true }}
```

actionlint (1.7.12)

[error] constant expression "true" in condition. remove the if: section

(if-cond)